### DISCUSSION TITLE
Looking for a nice default config

### DISCUSSION THREAD

**Discussion by glow12121** [JakeStanger/ironbar#349] [Q&A] [answered]
Hi,

Being a lazy user I am looking for a default Ironbar config that would look familiar and nice. Something like Windows XP or 7, or Budgie, or this:

![image](https://github.com/JakeStanger/ironbar/assets/29035180/1152e6b4-d8c8-4062-ad86-d91960375e88)

Is it possible to copy it from somewhere?

**Comment by JakeStanger** [comment, marked as answer]
Hey, the config inside the [examples](https://github.com/JakeStanger/ironbar/tree/master/examples) folder (also linked from the wiki) is based off the screenshot above. You will probably need to remove a few modules and move things around a bit, but the CSS should give you that design as-is.

**Reply by glow12121** [reply]
Thanks!
